x/authenticator/integration_test.go

```diff
@@ -315,26 +316,25 @@ func (s *AuthenticatorSuite) TestAuthenticatorStateExperiment() {
 		Amount:      sdk.NewCoins(sdk.NewInt64Coin(sdk.DefaultBondDenom, 1_000_000_000_000)),
 	}
 
-	stateful := testutils.StatefulAuthenticator{KvStoreKey: s.app.GetKVStoreKey()[authenticatortypes.StoreKey]}
+	stateful := testutils.StatefulAuthenticator{KvStoreKey: s.app.GetKVStoreKey()[authenticatortypes.ManagerStoreKey]}
 	s.app.AuthenticatorManager.RegisterAuthenticator(stateful)
 	err := s.app.AuthenticatorKeeper.AddAuthenticator(s.chainA.GetContext(), s.Account.GetAddress(), "Stateful", []byte{})
 	s.Require().NoError(err, "Failed to add authenticator")
 
 	// check account balances
 
 	_, err = s.chainA.SendMsgsFromPrivKeys(pks{s.PrivKeys[0]}, failSendMsg)
-	fmt.Println("err: ", err)
 	s.Require().Error(err, "Succeeded sending tx that should fail")
 
 	// Auth failed, so no increment
```

Comment:
we should prob check the lack of increment here

Reply:
I think it's checked two lines below

```go
	// Auth failed, so no increment
	s.Require().Equal(0, stateful.GetValue(s.chainA.GetContext()))
```